PsfLauncher/Readme.md

```diff
@@ -332,6 +335,13 @@ The PSF Launcher supports the use of two special purpose "pseudo-variables". The
 | %MsixPackageRoot% | The root folder of the package. While nominally this would be a subfolder under "C:\\Program Files\\WindowsApps" it is possible for the volume to be mounted in other locations. |
 | %MsixWritablePackageRoot% | The package specific redirection location for this user when the FileRedirectionFixup is in use. | 
 
+### InPackageContext
+This configuration allows all processes created when the executable is run to operate in the same package context. This includes any child processes that might not be part of the package. This might be useful in situations where applications span processes that are outside package but needs to be run in the same package context, or when PSF fixups need to be applied to processes external to the package.

```

Comment:
Can we change this description? 
This configuration allows all processes created when the executable is run to operate in the same package context. -->
This Configuration allows all created process from a running executable to operate in same package context.?